mmdet3d/datasets/custom_3d_seg.py

```diff
@@ -6,11 +6,13 @@
 from torch.utils.data import Dataset
 
 from mmdet.datasets import DATASETS
+from mmseg.datasets import DATASETS as SEG_DATASETS
 from .pipelines import Compose
 from .utils import get_loading_pipeline
 
 
 @DATASETS.register_module()
+@SEG_DATASETS.register_module()
```

Comment:
Need to register dataset twice because in `mmseg`'s `train_segmentor()`, it also calls `build_dataset`. This can be resolved in the future if we unify `DATASET` registry over all repos.